config/flags.go

```diff
@@ -0,0 +1,220 @@
+package config
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/spf13/pflag"
+	"github.com/spf13/viper"
+)
+
+type buxFlags struct {
+	pflag.FlagSet
+}
+
+type cliFlags struct {
+	showVersion bool `mapstructure:"version"`
+	showHelp    bool `mapstructure:"help"`
+	dumpConfig  bool `mapstructure:"dump_config"`
+}
+
+func loadFlags() error {
+	cli := cliFlags{}
```

Comment:
Kuba, it looks that you are right. 
I think it is optimized way to serve flags to viper.

I would suggest one improvement - small function like:
```go
func anyFlags() {
   if os.Args > 1 {
    return true
}

return false
}
```

We can call it in method `loadFlags() ` in line 22.